src/circles.rs

```diff
@@ -128,25 +138,38 @@ fn get_colored_circles (center_to_center: i32, small_radius: i32, colors: Vec<[u
     out
 }
 
-/// function to calculate contents of the png image with 19-circle icon
-pub fn calculate_png_data (big_radius: i32, colors: Vec<[u8; 4]>) -> Vec<u8> {
-    
+/// Calculate png image pixel data (only pixel colors)
+///
+/// Requires image size in pixels (equal to diameter of largest, outer circle),
+/// and identicon colors
+#[cfg(feature = "pix")]
+pub fn calculate_png_data(size_in_pixels: u16, colors: [[u8; 4]; 19]) -> Vec<u8> {
```

Comment:
```suggestion
pub fn calculate_png_data(size_in_pixels: u16, colors: [CircleColor; 19]) -> Vec<u8> {
```

and introduce a type alias